### DISCUSSION TITLE
how to show Intel(R) Movidius(TM) VPU usage in percentage like task manager performance

### DISCUSSION THREAD

**Discussion by yadavdix** [community/community#39627] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

using Microsoft .Net performance counter not able to to show VPU usage in % out of 100 like in task manager performance is showing. 

is there any way to show it using c++ or python.

**Comment by ossanna16** [comment]
Hi @yadavdix,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on .NET. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
